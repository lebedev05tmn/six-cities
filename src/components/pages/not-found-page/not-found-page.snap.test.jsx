import React from "react";
import {render} from "@testing-library/react";
import NotFoundPage from "./not-found-page";
import {BrowserRouter} from "react-router-dom";

test(`Should Not Found Page renders correctly`, () => {
  const {container} = render(
    <BrowserRouter>
      <NotFoundPage />
    </BrowserRouter>
  );
  expect(container).toMatchSnapshot();
});
