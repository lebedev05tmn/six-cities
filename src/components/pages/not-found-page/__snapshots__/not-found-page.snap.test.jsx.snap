// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Not Found Page renders correctly 1`] = `
<div>
  <div
    class="not-found"
    style="display: flex; flex-direction: column; align-items: center; justify-content: center; height: 50vh; font-size: 50px;"
  >
    <h1
      class="not-found__title"
      style="margin: 0px;"
    >
      404
    </h1>
    <p
      class="not-found__text"
    >
      Page not found
    </p>
    <a
      class="not-found__link"
      href="/"
      style="color: blue;"
    >
      Home Page
    </a>
  </div>
</div>
`;
